test/es6/regex-prototype-properties.js

```diff
@@ -79,5 +79,20 @@ tests = tests.concat(nonGenericPropertyNames.map(function (name) {
         }
     };
 }));
+tests = tests.concat([
+    {
+        name: "RegExp.prototype.toString should be generic",
+        body: function () {
+            var re = {
+                source: "source",
+                flags: "gi"
+            };
+
+            var string = RegExp.prototype.toString.call(re);

```

Comment:
What if the source and flags are not there, say RegExp.prototype.toString.call({}) ?